docs/apis-tools/administration-sm-api/administration-sm-api-authentication.md

```diff
@@ -1,14 +1,14 @@
 ---
-id: console-sm-api-authentication
+id: administration-sm-api-authentication
 title: "Authentication"
 sidebar_position: 2
-description: "Console SM API is a REST API and provides access to Console SM data. Requests and responses are in JSON notation."
+description: "The Administration API for Self-Managed is a REST API and provides access to Console Self-Managed data. Requests and responses are in JSON notation."
```

Comment:
If we use this sort of phrasing elsewhere in the docs, it might be good to standardize on `JSON` vs `JSON notation`, but this isn't blocking for this PR.